SCloudRandomizer/MusicSource.m

```diff
@@ -68,11 +67,11 @@ - (void)getTracks:(SearchParams *)searchParams completionHandler:(tracksFetchedC
     NSString *cleanedKeywords = [[searchParams.keywords stringByReplacingOccurrencesOfString:@" " withString:@"%20"] stringByReplacingOccurrencesOfString:@"," withString:@"%2C"];
     NSString *resourceURL = [NSString stringWithFormat:@"https://api.soundcloud.com/tracks?format=json&q=%@", cleanedKeywords];
     NSLog(@"The resourceURL is %@", resourceURL);
-    
-    [SCRequest performMethod:SCRequestMethodGET
+
+    [MusicSource fetchTracks:SCRequestMethodGET
```

Comment:
self can only be used with instance methods which makes sense since self points to a particular instance whereas class methods are associated with the class itself and not an instance